From b02058ece38f1f8d18d89509ef69b0ceb126db3e Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Thu, 27 Jun 2024 19:11:21 +0300
Subject: [PATCH] [SW-0] hlb325: release: remove srcs deployment

Avoid deploying the customer-shared sources during release, as this
target should be triggered directly by the release script.

Change-Id: I4a53ff28439ebbcadc4a4c53c4e315ff2b2ad429
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/scripts/make/hlb325/Makefile b/scripts/make/hlb325/Makefile
index e498b81..1c4ccdb 100644
--- a/scripts/make/hlb325/Makefile
+++ b/scripts/make/hlb325/Makefile
@@ -66,7 +66,7 @@
 		$(IMAGES_DIR)
 
 .PHONY: hlb325-release
-hlb325-release: hlb325-check-release hlb325-check-branch hlb325-deploy-srcs
+hlb325-release: hlb325-check-release hlb325-check-branch
 	$(call hl-release-eval,$(patsubst %-release,%,$@))
 	$(call hlb325-release-push,$(patsubst %-release,%,$@))
 	$(call hlb325-release-test,$(patsubst %-release,%,$@))
